pubspec.yaml

```diff
@@ -22,7 +22,7 @@ dependencies:
   chewie: '>=1.1.0 <2.0.0'
 
   # Plugin for rendering the <iframe> tag.
-  webview_flutter: '>=2.0.4 <3.0.0'
+  webview_flutter: '>=2.0.4 <=3.0.0'
```

Comment:
```yaml
  webview_flutter: '>=2.0.4'
```

Reply:
Instead of the above, I would suggest 

```yaml
webview_flutter: '>=2.0.4 <4.0.0'
```

Reply:
Changed to `webview_flutter: '>=2.0.4 <4.0.0'` as per the suggestion of @tneotia